src/ConnectAuth.ts

```diff
@@ -0,0 +1,192 @@
+import { AuthError, AuthErrorCode } from './AuthError';
+import { decodeUnverifiedConnectJwt, verifyConnectJwt } from './Jwt';
+import { RequestReader } from './requestReader';
+import { ConnectJwt, ContextQsh, CredentialsLoader, InstallType } from './types';
+
+interface CommonVerifyArgs<E> {
+  baseUrl: string;
```

Comment:
I have made the verification methods static so baseUrl is passed to the methods. I'm avoiding keeping state on the main class as it does not bring much advantage and only makes the code more coupled by sharing a state (`this`).